build-system/internal-version.js

```diff
@@ -30,7 +30,7 @@ exports.TOKEN = getToken();
 
 function getToken() {
   var task = process.argv[2];
-  if (!task || task == 'build') {
+  if (!task || task == 'build' || task == 'test') {
```

Comment:
This is a white list to make sure we don't go in here by accident. `dist` wants the production path.